static/sidebar/gruntfile.js

```diff
@@ -37,11 +40,11 @@ module.exports = function(grunt) {
         }
       },
       uglify: {
-        options: {
```

Comment:
Please revert this change, we will check if this does not work for everyone first and if necessary we will need to find a different package to do this task